velox/connectors/Connector.h

```diff
@@ -345,7 +346,9 @@ class Connector {
       RowTypePtr inputType,
       std::shared_ptr<ConnectorInsertTableHandle> connectorInsertTableHandle,
       ConnectorQueryCtx* connectorQueryCtx,
-      CommitStrategy commitStrategy) = 0;
+      CommitStrategy commitStrategy,
+      tsan_atomic<bool>* nonReclaimableSection,
```

Comment:
I think each hive data sink needs its own flag so let's don't pass this flag from writer operator? thanks!